File: src/main/java/com/eager/questioncloud/domain/question/repository/QuestionCategoryJpaRepository.java
package com.eager.questioncloud.domain.question.repository;

import com.eager.questioncloud.domain.question.entity.QuestionCategoryEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface QuestionCategoryJpaRepository extends JpaRepository<QuestionCategoryEntity, Long> {
}

File: qc-core/src/main/java/com/eager/questioncloud/core/domain/question/infrastructure/repository/QuestionCategoryJpaRepository.kt
package com.eager.questioncloud.core.domain.question.infrastructure.repository;

import com.eager.questioncloud.core.domain.question.infrastructure.entity.QuestionCategoryEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface QuestionCategoryJpaRepository extends JpaRepository<QuestionCategoryEntity, Long> {
}
